nwbib/nwbib.ttl

```diff
@@ -81,14 +81,17 @@
     skos:broader :N100000 ;
     skos:inScheme <https://nwbib.de/subjects> ;
     skos:notation "100100" ;
-    skos:prefLabel "Allgemeine Landeskunde - Allgemeines"@de .
+    skos:prefLabel "Allgemeine Landeskunde - Allgemeines"@de ;
+    skos:closeMatch gnd:4020216-1 .
```

Comment:
Ist das nicht eher https://d-nb.info/gnd/4073972-7 ?

Reply:
ja, das stimmt. Wie kann ich das nochmal ändern?

Reply:
Bezieht sich die Frage darauf, wie du Änderungen in dem Branch `204-addSkoselementsforNWBibMALIS-Projekt` vornehmen kannst?

Reply:
Ja genau, ich habe das zu lang nicht mehr gemacht. 😄 Außerdem gäbe es dann noch die Änderung für Schiffahrt mit einem und zwei "f" und Photographie / Fotographie, die ich als altLabel eintragen möchte. Da hatten wir, Adrian, am 16.2. per Mail geschrieben. Es ging um die Auffindbarkeit beider/aller Schreibweisen bei der Suche im NWBib-Sachsystematik-Suchschlitz unter https://nwbib.de/subjects . Das kann ich aber sonst auch in einem neuen Branch machen. LG

Reply:
Du hattest den Prozess ja selbst mal hier dokumentiert: https://service-wiki.hbz-nrw.de/x/UID8Ng